src/helpers/workspaces/workspaces-helper.ts

```diff
@@ -3,5 +3,54 @@ import { getWorkspacesApi } from './api';
 const workspacesApi = getWorkspacesApi();
 
 export async function getWorkspaces() {
-  return await workspacesApi.listWorkspaces();
+  return await workspacesApi.listWorkspaces().then((props: any) => ({
+    ...props,
+    data: [
+      {
+        uuid: '3fa85f64-5717-4562-b3fc-2c963f66afa6',
+        name: 'Workspace A',
+        description: 'Description of Workspace A',
+        created: '2024-08-29T08:16:59.959Z',
+        modified: '2024-08-29T08:16:59.959Z',
+      },
+      {
+        uuid: 'a1e9652c-4d9e-453f-b6bd-bc702945f4a6',
+        parent: '3fa85f64-5717-4562-b3fc-2c963f66afa6',
+        name: 'Workspace B',
+        description: 'Description of Workspace A',
+        created: '2024-08-29T08:16:59.959Z',
+        modified: '2024-08-29T08:16:59.959Z',
+      },
+      {
+        uuid: 'ab69e6e2-cdc3-4d75-9c82-3ef256c5c7bf',
+        parent: '3fa85f64-5717-4562-b3fc-2c963f66afa6',
+        name: 'Workspace C',
+        description: 'Description of Workspace A',
+        created: '2024-08-29T08:16:59.959Z',
+        modified: '2024-08-29T08:16:59.959Z',
+      },
+      {
+        uuid: '50070ed8-3715-4927-838e-63bd63f69aeb',
+        name: 'Root Workspace B',
+        description: 'Description of Workspace A',
+        created: '2024-08-29T08:16:59.959Z',
+        modified: '2024-08-29T08:16:59.959Z',
+      },
+    ],
+  }));
+}
+
+export async function getWorkspace(ws: string) {
+  return await workspacesApi.getWorkspace({ uuid: ws }).then((props: any) => ({
+    ...props,
+    data: {
+      uuid: '3fa85f64-5717-4562-b3fc-2c963f66afa6',
+      parent_id: '3fa85f64-5717-4562-b3fc-2c963f66afa6',
+      type: 'root',
+      name: 'Workspace A',
+      description: 'Description of Workspace A',
+      created: '2024-11-06T12:06:36.563Z',
+      modified: '2024-11-06T12:06:36.563Z',
+    },
+  }));
```

Comment:
No need to commit mocked data
```suggestion
  return workspacesApi.getWorkspace({ uuid: ws });
```

Reply:
Fixed